online_carolapp/.editorconfig

```diff
@@ -0,0 +1,44 @@
+# https://editorconfig.org/
```

Comment:
do we need this file?
maybe add to .gitignore?

Reply:
I think so, take a look:

> EditorConfig helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs. The EditorConfig project consists of a file format for defining coding styles and a collection of text editor plugins that enable editors to read the file format and adhere to defined styles. EditorConfig files are easily readable and they work nicely with version control systems.

It helps us to make the consistency cross editors.

Keeping the file.